modules/mini-games/tightspot.lua

```diff
@@ -0,0 +1,758 @@
+            ent.destroy()
+        end
+    end
+    for i, entity in ipairs(save["entity"]) do
+        local name = entity[1]
+        local position = entity[2]
+        local force = entity[3]
+        local minable = entity[4]
+        position.x = position.x
+        entity[2].x = position.x
+        local ent = variables["surface"].create_entity{name = name , position = position , force = force }
+        ent.minable = minable
+    end
+    local players = {}
+    for name,_ in pairs(centers) do
+        players[#players+1] = name
+    end
+    local scores = {}
+    for _,name in ipairs(players) do
+        local player = game.players[name]
+        scores[#scores+1] = {Store.get(balances,player),name}
+    end
+    local colors =  {
+        ["1st"] = "[color=#FFD700]",
+        ["2nd"] = "[color=#C0C0C0]",
+        ["3rd"] = "[color=#cd7f32]"
+    }
+    table.sort(scores,function(a,b) return a[1] > b[1] end)
+    for i, score in ipairs(scores) do
+        local money = score[1]
+        local player_name = score[2]
+        local place = Nth(i)
+        if colors[place] then
+            game.print(colors[place]..place..": "..player_name.." with "..money.." points.[/color]")
+        else
+            game.print("[color=#808080]"..place..": "..player_name.." with "..money.." points.[/color]")
+        end
+    end
+    reset_table(centers)
+    reset_table(markets)
+    reset_table(entities)
+    reset_table(started)
+    reset_table(chests)
+    reset_table(variables)
+    reset_table(left_players)
+end
+
+local function placed_entety(event)
+    local entity = event.created_entity
+    local position = entity.position
+    local player = game.players[event.player_index]
+    if entity.type ~= "wall" then
+        entities[entity.name] = entity.name
+        entity.active = false
+    else
+        if not variables.walls[player.name][position.x..','..position.y] then
+            player.print("You cant resell this land.")
+            player.insert{name = entity.name, count = 1}
+            entity.destroy()
+        else
+            local price = land_price(player,position)
+            change_balance(player,price)
+            player.surface.create_entity {
+                name = "flying-text",
+                position = position,
+                text = "+" .. price,
+                color = {g = 1 }
+            }
+        end
+    end
+end
+local token_for_replace_wall
+local function mined(event)
+    local entity = event.entity
+    if entity.type == "wall" then
+        local position = entity.position
+        local player = game.players[event.player_index]
+        if not variables.walls[player.name][position.x..','..position.y] then
+            local force = entity.force
+            task.set_timeout_in_ticks(1, token_for_replace_wall,{{name = entity.name , position = position , force = force},player.name})
+            player.print("How did you get here.")
```

Comment:
If this isnt meant to happen you might want to log this event, or at least tell the player to tell an admin.

Reply:
its an easter egg i gues as it not possible without commands